### PR TITLE
Modernize

### PR DESCRIPTION
**Pull request by luis-pereira**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -39,16 +39,16 @@
...
-    {"drop",    0, NULL, 'd'},
-    {"profile", 1, NULL, 'p'},
-    {NULL,      0, NULL,  0}
+    {"version", 0, nullptr, 'v'},
+    {"help",    0, nullptr, 'h'},
+    {"workdir", 1, nullptr, 'w'},
+    {"execute", 1, nullptr, 'e'},
+    {"drop",    0, nullptr, 'd'},
+    {"profile", 1, nullptr, 'p'},
+    {nullptr,      0, nullptr,  0}
```
Comment (yan12125) [inline src/main.cpp]: Minor style issue: this line is no longer aligned with previous lines.

Comment (luis-pereira) [reply src/main.cpp]: Nice catch. Will fix it.

**Comment by luis-pereira** [comment]
@yan12125 Done.

**Comment by yan12125** [comment]
Gonna merge - I don't believe this PR will cause an issue.

**Pull request merged**